server/polar/customer_portal/schemas/order.py

```diff
@@ -61,3 +61,33 @@ class CustomerOrderInvoice(Schema):
 
 class CustomerOrderUpdate(OrderUpdateBase):
     """Schema to update an order."""
+
+
+class CustomerOrderPaymentStatus(Schema):
+    """Payment status for an order."""
+
+    status: str = Field(..., description="Current payment status.")
+    error: str | None | None = Field(
+        None, description="Error message if payment failed."
+    )
+
+
+class CustomerOrderConfirmPayment(Schema):
+    """Schema to confirm a retry payment using a Stripe confirmation token."""
+
+    confirmation_token_id: str = Field(
```

Comment:
Should we add a `payment_processor: PaymentProcessor` field? So the API can be somewhat future-proof if we add other processors.